packages/block-editor/src/components/block-toolbar/block-toolbar-icon.js

```diff
@@ -57,21 +56,19 @@ function getBlockIconVariant( { select, clientIds } ) {
 
 	const isDefaultEditingMode =
 		getBlockEditingMode( clientIds[ 0 ] ) === 'default';
-	const _hideTransformsForSections =
```

Comment:
Removed the `_` as this is only used when we have a `const` naming conflict. These aren't exposed from the selector, so we can use the default naming.